src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -25,7 +25,7 @@ public function assertResponseOk()
 	 */
 	public function assertResponseStatus($code)
 	{
-		return PHPUnit::assertEquals($code, $this->response->getStatusCode());
+		return PHPUnit::assertEquals($code, $this->response->getStatusCode(), 'Expected status code '.$code.', got '.$actual);
```

Comment:
Wait until we get some complex objects interpolated into strings, then you won't sound so funny :) I will do it as you ask (noticed inside the same file, down below, everything is inside the string, instead of being concat'ed).